met-api/tests/unit/api/test_user.py

```diff
@@ -87,5 +90,26 @@ def test_get_staff_users_exclude_public_user(client, jwt, session, ):  # pylint:
     assert staff_user is not None
 
 
+def test_add_user_to_admin_group(mocker, client, jwt, session):  # pylint:disable=unused-argument
+    """Assert that you can add a user to admin group."""
+    user = factory_user_model()
+
+    mock_add_user_to_group_keycloak_response = MagicMock()
+    mock_add_user_to_group_keycloak_response.status_code = HTTPStatus.NO_CONTENT
+    mock_add_user_to_group_keycloak = mocker.patch(
+        'met_api.services.keycloak.KeycloakService.add_user_to_group',
+        return_value=mock_add_user_to_group_keycloak_response
+    )
+
```

Comment:
there is actual keycloak instance for tests..so we can get away from mocking and test real scenarios if you like that..this test doesnt have to change

but an actual integration test to see adding user to admin and then fetching the users role back and asserting admin will be a nice workflow...